test/integration/freetext_editor_spec.mjs

```diff
@@ -1049,11 +1064,19 @@ describe("FreeText Editor", () => {
           // canvas.
           editorIds = await getEditors(page, "freeText");
           expect(editorIds.length).withContext(`In ${browserName}`).toEqual(1);
-          const hidden = await page.$eval(
-            "[data-annotation-id='26R'] canvas",
-            el => getComputedStyle(el).display === "none"
-          );
-          expect(hidden).withContext(`In ${browserName}`).toBeTrue();
+
+          const isEditorWhite = await page.evaluate(rect => {
+            const canvas = document.querySelector(".canvasWrapper canvas");
+            const ctx = canvas.getContext("2d");
+            const { data } = ctx.getImageData(
+              rect.x,
+              rect.y,
+              rect.width,
+              rect.height
+            );
+            return data.every(x => x === 0xff);
+          }, editorRect);
```

Comment:
Rather than repeating *almost* the same code again here, would it make sense to add a helper function instead?